exercises/nucleotide-count/README.md

```diff
@@ -1,34 +1,23 @@
 # Nucleotide Count
 
-Given a single stranded DNA string, compute how many times each nucleotide occurs in the string. Also if the
-given input strand contains invalid nucleotides, throw the error "Invalid nucleotide in strand"
+Given a single stranded DNA string, compute how many times each nucleotide occurs in the string.
```

Comment:
we throw an error for `invalid nucleotide`. This testcase needs to be removed: https://github.com/exercism/javascript/blob/master/exercises/nucleotide-count/nucleotide-count.spec.js#L21

Reply:
Are you asking to have nucleotide-count/README.md changes removed from this PR?

Reply:
This was a note-to-self point to be done in another PR. Sorry for the confusion.